src/components/dialog/dialog.component.tsx

```diff
@@ -181,9 +185,16 @@ export const Dialog = forwardRef<DialogHandle, DialogProps>(
         midPointX = 0;
       }
 
+      if (size === "maximise") {
+        const breakPoint = window.innerWidth > 960 ? 32 : 16;
```

Comment:
suggestion: for consistency sake might be worth storing these values in `const`s as that's what has been done elsewhere in this file etc